frictionless/errors/row.py

```diff
@@ -66,11 +66,72 @@ class PrimaryKeyError(RowError):
     template = 'Row at position "{rowNumber}" violates the primary key: {note}'
 
 
-class ForeignKeyError(RowError):
+@attrs.define(kw_only=True)
+class ForeignKeyError(TableError):
     type = "foreign-key"
     title = "ForeignKey Error"
     description = "Values in the foreign key fields should exist in the reference table"
     template = 'Row at position "{rowNumber}" violates the foreign key: {note}'
+    tags = ["#table", "#row"]
```

Comment:
We don't need to duplicate `tags` and `cells` here in the class definition as it's inherited from the `RowError` class